kitsune/questions/apps.py

```diff
@@ -1,6 +1,16 @@
 from django.apps import AppConfig
 
 
+def is_installed(model_class):
+    """
+    This is a fix for django-activity-stream 1.4.0 that
+    should fix an issue. Evidently django-activity-stream
+    will have a release just for 4.1 at some point...
+    https://github.com/justquick/django-activity-stream/issues/515
+    """
+    return model_class._meta.app_config is not None
```

Comment:
Let's capture an issue to clean this up when a new release is out and move it under the `blocked` column

Reply:
https://github.com/mozilla/sumo/issues/1143

Reply:
Looks like you started the issue already - so, agreed!